file/reader_test.go

```diff
@@ -99,12 +99,13 @@ func TestReadKongStateFromStdin(t *testing.T) {
 	os.Stdin = tmpfile
 
 	c, err := GetContentFromFiles(filenames)
-	assert.NotNil(c)
 	assert.Nil(err)
 
-	assert.Equal(kong.Service{
-		Name: kong.String("test service"),
-		Host: kong.String("test.com"),
-	},
-		c.Services[0].Service)
+	if assert.NotNil(c) && assert.NotEmpty(c.Services) {
+		assert.Equal(kong.Service{
+			Name: kong.String("test service"),
+			Host: kong.String("test.com"),
+		},
+			c.Services[0].Service)
```

Comment:
Why this diff? I'm sure there is a good reason for this but I can't see why this is a conditional assert. Shouldn't these tests be deterministic?